modules/installation/pages/proxy-setup.adoc

```diff
@@ -129,35 +127,35 @@ Activate advertising proxy via SLP?::
 SLP stands for Service Location Protocol.
 
 If parts are missing, such as CA key and public certificate, the script prints commands that you must execute to integrate the needed files.
-When the mandatory files are copied, re-run [command]``configure-proxy.sh``.
-Also restart the script if a HTTP error was met during script execution.
+When the mandatory files are copied, run [command]``configure-proxy.sh`` again.
+Also run again the script if a HTTP error was met during script execution.
 
 [command]``configure-proxy.sh`` activates services required by {productname} Proxy, such as [systemitem]``squid``, [systemitem]``apache2``, [systemitem]``salt-broker``, and [systemitem]``jabberd``.
 
-To check the status of the proxy system and its clients, click the proxy system's details page on the {webui} (menu:Main Menu[Systems > Proxy], then the system name).
-[guimenu]``Connection`` and [guimenu]``Proxy`` subtabs display the respective status information.
+To check the status of the proxy system and its clients, click the proxy system's details page on the {webui} (menu:Systems[Proxy], then the system name).
+[guimenu]``Connection`` and [guimenu]``Proxy`` subtabs display various status information.
 
 
 
 
 [[proxy.pxe.setup]]
-== Enabling PXE Boot via {susemgrproxy}
+== Enable PXE Boot through {susemgrproxy}
 
 
 
 [[proxy.pxe.sync]]
-=== Synchronizing Profiles and System Information
+=== Synchronize Profiles and System Information
 
-To enable PXE boot via a proxy server, additional software must be installed and configured on both the {productname} server and the {susemgrproxy} server.
+To enable PXE boot through a proxy server, additional software must be installed and configured on both the {productname} Server and the {susemgrproxy} Server.
 
-. On the {productname} server install [package]#susemanager-tftpsync# :
+. On the {productname} Server install [package]#susemanager-tftpsync#:
 +
 
 ----
 zypper in susemanager-tftpsync
 ----
 
-. On the {susemgrproxy} server install [package]#susemanager-tftpsync-recv# :
+. On the {susemgrproxy} Server install [package]#susemanager-tftpsync-recv#:
```

Comment:
SUSE Manager Proxy Server?

Reply:
We sometimes use (or used) to call the proxy a "proxy server"--this is probably confusing and we should stop it... @Loquacity

Reply:
yeah, it doesn't take much to miss the word 'proxy' in there.

Reply:
ok, please review again.